tests/testutils/pdata_to_resource_metrics.go

```diff
@@ -85,7 +79,7 @@ func addDoubleSum(ilms *InstrumentationLibraryMetrics, metric pdata.Metric) {
 	}
 	for l := 0; l < doubleSum.DataPoints().Len(); l++ {
 		dp := doubleSum.DataPoints().At(l)
-		val := dp.Value()
+		val := dp.DoubleVal()
```

Comment:
I think this is `dp.Type()` dependent and `IntVal()` is needed for `MetricValueTypeInt`.

Reply:
Done.